ruddock/modules/rotation/helpers.py

```diff
@@ -117,3 +118,16 @@ def change_bucket(prefrosh_id, new_bucket_name):
     WHERE prefrosh_id = (:pid)
     """)
   flask.g.db.execute(query, b=new_bucket_name, pid=prefrosh_id)
+
+def escape_comments(str):
+  if str is not None:
+    # cgi.escape escapes '<', '>', and '&'
+    # unicode.escape escapes newlines to '\n'
+    # the replace statement escapes apostrophes
+    # TODO: refactor this to something less hacky
+    ret = str.encode('unicode_escape')
+    ret = ret.replace("'", "\\'")
+    ret = cgi.escape(ret)
```

Comment:
the jinja2 templating engine already handles all html escaping to prevent XSS attacks. I don't think this is necessary?